packages/textInput/components/TextInputWithButtons.tsx

```diff
@@ -86,12 +95,23 @@ const TextInputWithButtons = (props: TextInputWithButtonsProps) => {
                 getInputAppearanceStyle(inputAppearance)
               )}
             >
-              {getIconStartContent(props, getInputAppearance())}
+              {getIconStartContent(
+                {
+                  type,
```

Comment:
I think this is missing some props. At least for `iconStart`, as the icon is no longer showing up in storybook.